packages/styles/src/core/layout/toolbar/_actions.scss

```diff
@@ -2,14 +2,22 @@
   display: flex;
   position: absolute;
   z-index: 3;
-  top: $header-height;
+  top: $toolbar-size;
   right: 0;
   bottom: 0;
   left: -100%;
   flex-direction: column;
+  align-items: flex-start;
   width: 100%;
   transition: left 0.3s;
-  background-color: var(--brand-02-color, $brand-02-color);
+  background-color: var(--toolbar-tabs-bg-color, $brand-02-color);
```

Comment:
This won't work on `IE11`, we need a fallback:

```
background-color: $brand-02-color;
background-color: var(--toolbar-tabs-bg-color, $brand-02-color);
```

Reply:
Yes, we need a fallback. Maybe a PostCSS plugin?

Reply:
yep, we can wait for the first release to do this kind of fine-tunning